cirq/__init__.py

```diff
@@ -61,6 +61,7 @@
     dot,
     expand_matrix_in_orthogonal_basis,
     hilbert_schmidt_inner_product,
+    eye_tensor,
```

Comment:
You can pull this into a separate PR while we dither over the details of this one.

Reply:
I won't bother unless you want it for something.